### ISSUE TITLE
Building feature/offline Error

### ISSUE THREAD

**Issue by mcornock** [njoy/ENDFtk#145]
```cmake ../ -DFETCHCONTENT_BASE_DIR=../dependencies
-- Using release repositories
-- 
-- -----------------------------------------------------------
-- 
-- catch-adapter
-- Git current branch: HEAD
-- Git commit hash: fb84b82ebf7a4789aa43cea560680cf745c6ee4f
-- 
-- -----------------------------------------------------------
-- 
-- -----------------------------------------------------------
-- 
-- hana-adapter
-- Git current branch: HEAD
-- Git commit hash: f58e8973c9a614dc4f3720b5581a762c61bdbb40
-- 
-- -----------------------------------------------------------
-- pybind11 v2.6.1 
CMake Warning at dependencies/range-v3-src/cmake/ranges_env.cmake:77 (message):
  [range-v3 warning]: unknown build type, defaults to release!
Call Stack (most recent call first):
  dependencies/range-v3-src/CMakeLists.txt:50 (include)


-- Found Doxygen: /usr/bin/doxygen (found version "1.8.5") found components: doxygen dot 
-- [range-v3]: C++ std=1z
-- Performing Test RANGES_HAS_CXXSTD
CMake Error in /workspace/ENDFtk-offline/bin/CMakeFiles/CMakeTmp/CMakeLists.txt:
  Target "cmTC_657e6" requires the language dialect "CXX17" (with compiler
  extensions), but CMake does not know the compile flags to use to enable it.


CMake Error at /cmake/3.19.1/share/cmake-3.19/Modules/Internal/CheckSourceCompiles.cmake:89 (try_compile):
  Failed to generate test project build system.
Call Stack (most recent call first):
 /cmake/3.19.1/share/cmake-3.19/Modules/Internal/CheckCompilerFlag.cmake:66 (cmake_check_source_compiles)
  /cmake/3.19.1/share/cmake-3.19/Modules/CheckCXXCompilerFlag.cmake:40 (cmake_check_compiler_flag)
  dependencies/range-v3-src/cmake/ranges_flags.cmake:16 (check_cxx_compiler_flag)
  dependencies/range-v3-src/cmake/ranges_flags.cmake:86 (ranges_append_flag)
  dependencies/range-v3-src/CMakeLists.txt:51 (include)


-- Configuring incomplete, errors occurred!
See also "/workspace/ENDFtk-offline/bin/CMakeFiles/CMakeOutput.log".```

**Comment by mcornock** [comment]
```
cmake ../ -DFETCHCONTENT_BASE_DIR=../dependencies
-- The CXX compiler identification is GNU 9.2.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /gcc/9.2.0/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Using release dependencies
-- Found PythonInterp: /Anaconda/bin/python (found version "3.8.3") 
-- Found Git: /git/2.31.1/bin/git (found version "2.31.1") 
-- 
-- -----------------------------------------------------------
-- 
-- catch-adapter
-- Git current branch: HEAD
-- Git commit hash: fb84b82ebf7a4789aa43cea560680cf745c6ee4f
-- 
-- -----------------------------------------------------------
-- 
-- -----------------------------------------------------------
-- 
-- hana-adapter
-- Git current branch: HEAD
-- Git commit hash: f58e8973c9a614dc4f3720b5581a762c61bdbb40
-- 
-- -----------------------------------------------------------
-- pybind11 v2.6.1 
-- Found PythonLibs: /Anaconda/conda3/envs/python3.8.3/lib/libpython3.8.so
-- Performing Test HAS_FLTO
-- Performing Test HAS_FLTO - Success
CMake Warning at dependencies/range-v3-src/cmake/ranges_env.cmake:77 (message):
  [range-v3 warning]: unknown build type, defaults to release!
Call Stack (most recent call first):
  dependencies/range-v3-src/CMakeLists.txt:50 (include)


-- Found Doxygen: /bin/doxygen (found version "1.8.5") found components: doxygen missing components: dot
-- [range-v3]: C++ std=1z
-- Performing Test RANGES_HAS_CXXSTD
-- Performing Test RANGES_HAS_CXXSTD - Success
-- Performing Test RANGES_HAS_WALL
-- Performing Test RANGES_HAS_WALL - Success
-- Performing Test RANGES_HAS_WEXTRA
-- Performing Test RANGES_HAS_WEXTRA - Success
-- Performing Test RANGES_HAS_WERROR
-- Performing Test RANGES_HAS_WERROR - Success
-- Performing Test RANGES_HAS_FDIAGNOSTIC_SHOW_TEMPLATE_TREE
-- Performing Test RANGES_HAS_FDIAGNOSTIC_SHOW_TEMPLATE_TREE - Success
-- Performing Test RANGES_HAS_FTEMPLATE_BACKTRACE_LIMIT
-- Performing Test RANGES_HAS_FTEMPLATE_BACKTRACE_LIMIT - Success
-- Performing Test RANGES_HAS_FMACRO_BACKTRACE_LIMIT
-- Performing Test RANGES_HAS_FMACRO_BACKTRACE_LIMIT - Failed
-- Performing Test RANGES_HAS_OMIT_FRAME_POINTER
-- Performing Test RANGES_HAS_OMIT_FRAME_POINTER - Success
-- Performing Test RANGES_HAS_OFAST
-- Performing Test RANGES_HAS_OFAST - Success
-- Performing Test RANGES_HAS_STRICT_ALIASING
-- Performing Test RANGES_HAS_STRICT_ALIASING - Success
-- Performing Test RANGES_HAS_STRICT_VTABLE_POINTERS
-- Performing Test RANGES_HAS_STRICT_VTABLE_POINTERS - Failed
-- Performing Test RANGES_HAS_FAST_MATH
-- Performing Test RANGES_HAS_FAST_MATH - Success
-- Performing Test RANGES_HAS_VECTORIZE
-- Performing Test RANGES_HAS_VECTORIZE - Failed
-- Performing Test RANGES_HAS_SIZED_DEALLOCATION
-- Performing Test RANGES_HAS_SIZED_DEALLOCATION - Success
-- Performing Test RANGES_HAS_MARCH_NATIVE
-- Performing Test RANGES_HAS_MARCH_NATIVE - Success
-- Performing Test RANGES_HAS_MTUNE_NATIVE
-- Performing Test RANGES_HAS_MTUNE_NATIVE - Success
-- Performing Test RANGE_V3_ALIGNED_NEW_PROBE
-- Performing Test RANGE_V3_ALIGNED_NEW_PROBE - Success
-- Performing Test RANGE_V3_HAS_FCONCEPTS
-- Performing Test RANGE_V3_HAS_FCONCEPTS - Success
-- Adding ENDFtk unit testing
-- Configuring done
-- Generating done
-- Build files have been written to: /workspace/ENDFtk-offline/build

$ cmake ../ -DLOCAL_BUILD=ON -DCMAKE_BUILD_TYPE=Release
-- The CXX compiler identification is GNU 9.2.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /gcc/9.2.0/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Using local dependencies
CMake Error at /cmake/3.19.1/share/cmake-3.19/Modules/ExternalProject.cmake:1488 (message):
  error: do not know how to extract
  '/workspace/ENDFtk-offline/dependencies/catch-adapter-src'
  -- known types are .7z, .tar, .tar.bz2, .tar.gz, .tar.xz, .tbz2, .tgz, .txz
  and .zip
Call Stack (most recent call first):
  /cmake/3.19.1/share/cmake-3.19/Modules/ExternalProject.cmake:2739 (_ep_write_extractfile_script)
  /cmake/3.19.1/share/cmake-3.19/Modules/ExternalProject.cmake:3430 (_ep_add_download_command)
  CMakeLists.txt:13 (ExternalProject_Add)


-- Configuring incomplete, errors occurred!
See also "/workspace/ENDFtk-offline/build/_deps/catch-adapter-subbuild/CMakeFiles/CMakeOutput.log".

CMake Error at /cmake/3.19.1/share/cmake-3.19/Modules/FetchContent.cmake:977 (message):
  CMake step for catch-adapter failed: 1
Call Stack (most recent call first):
 /cmake/3.19.1/share/cmake-3.19/Modules/FetchContent.cmake:1111:EVAL:2 (__FetchContent_directPopulate)
 /cmake/3.19.1/share/cmake-3.19/Modules/FetchContent.cmake:1111 (cmake_language)
  /cmake/3.19.1/share/cmake-3.19/Modules/FetchContent.cmake:1154 (FetchContent_Populate)
  cmake/local_dependencies.cmake:45 (FetchContent_MakeAvailable)
  CMakeLists.txt:44 (include)


-- Configuring incomplete, errors occurred!
See also "/workspace/ENDFtk-offline/build/CMakeFiles/CMakeOutput.log".
```

**Comment by whaeck** [comment]
@mcornock I know this is an old issue but I assume this can be closed? BTW: we're releasing ENDFtk v0.5.0 soon which is the full ENDF capability.
